assessments/guide/security-reviewer.md

```diff
@@ -0,0 +1,31 @@
+# Security Reviewers
+
+Reviewers will try to understand the
+system and probe its security.  Specifically design level issues are meant
+to be addressed as well as high level problems with the project’s setup and
+operation. This is meant to provide an independent analysis and estimation
+of the above information.  The goal is to ask questions to understand if
+there are hidden assumptions, underestimated risk, or design issues that
+harm the security of the project.  It may be useful to reach out to
+community members to understand the answers to some questions, especially
+involving deployment scenarios and the impact of attacks.
```

Comment:
Perhaps there should be a formal "request for comment" process where domain experts are solicited and there is a waiting period to allow those experts to weigh in?  eg 10 days?  perhaps we should put the burden on the project under review to nominate 2-3 non-affiliated subject matter experts to participate and advise the reviewer team?

Reply:
This makes sense but I think might be a little too heavyweight for the initial iterations of this process.  We will grow and add to this over time in the most helpful ways possible and this certainly could be one of those directions.

Reply:
that is specified in outline.md "Project posts their document to the group mailing list, allowing at least
   one week for review before presenting to the WG"